File: Api/Reservationpitch.Application/DTOs/UserDtos/Request/UpdateUserRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Reservationpitch.Application.DTOs.UserDtos.Request
{
    public class UpdateUserRequestDto
    {
        public string Name { get; set; } = string.Empty;
        public string PhoneNumber { get; set; } = string.Empty;
    }
}

File: App/stadium_reservation/lib/models/Dtos/Auth/updateuserrequestdto.dart
class Updateuserrequestdto {
  String name;
  String phoneNumber;

  Updateuserrequestdto({required this.name, required this.phoneNumber});

  Map<String, dynamic> toJson() {
    return {'name': name, 'phoneNumber': phoneNumber};
  }
}
